nautobot/docs/models/extras/dynamicgroup.md

```diff
@@ -1,7 +1,166 @@
 
-When creating a Dynamic Group, one must select a Content Type to which it is associated, for example `dcim.device`. The filtering parameters saved to the group behave as a bi-directional search query that used to identify members of that group, and can also be used to determine from an individual object in which Dynamic Groups it is a member.
+When creating a Dynamic Group, one must select a Content Type to which it is associated, for example `dcim.device`. The filtering parameters saved to the group behave as a bi-directional search query that used to identify members of that group, and can also be used to determine from an individual object the list of Dynamic Groups to which it belongs.
 
 Once created the Content Type for a Dynamic Group may not be modified as this relationship is tightly-coupled to the available filtering parameters. All other fields may be updated at any time.
+
+## Basic Filtering
+
+Dynamic Groups filtering is powered by **FilterSet** objects underneath the hood. Basic filtering is performed using the `filter` that is defined on a given Dynamic Group.
+
+An object is considered to be a member of a Dynamic Group if it is of the same Content Type and it is not excluded by way of any of the filter critera specified for that group. By default, if a group has an empty filter (`{}`) it will include all objects of the matching Content Type, just as a defaut list view of objects would prior to any filter fields being filled in the web UI.
+
+For example, for a Dynamic Group with Content Type of `dcim.device` and an empty filter, the list of members would be equivalent to the queryset for `Device.objects.all()`.
+
+!!! warning
+    This behavior was changed in v1.4.0. In v1.3.0 the default for a group with an empty filter was to fail "closed" and have zero members. As of v1.4.0, this behavior has been inverted to include all objects matching the content type by default instead of matching no objects. This was necessary to implement the progressive layering of child filters similarly to how we use filters to reduce desired objects from basic list view filters. This will described in more detail below.
+
+The Dynamic Group edit view has a **Filter Fields** tab that allows one to specify filter criteria. The filter fields available for a given Content Type are backed and validated by underlying filterset classes (for example `nautobot.dcim.filters.DeviceFilterSet`) and are represented in the web interface as a dynamically-generated filter form that corresponds to each eligible filter field.
+
+Any invalid field names that are not eligible for filtering objects will be discarded upon validation. Any invalid field values will result in a validation error.
```

Comment:
I need to make sure that as of v1.4 that this assertion is still correct. I believe that strict filtering may in face make the filterset complain loudly on an invalid filter parameter.

Reply:
Yep. This assertion is no longer correct. Mention needs to be made of the new `settings.STRICT_FILTERING` option, otherwise invalid filter parameters do result in errors:

```python
In [3]: s = parent.filterset_class({"bogus": True}, Device.objects.all())

In [4]: s.is_valid()
Out[4]: False

In [5]: s.errors
Out[5]: {'bogus': ['Unknown filter field']}
```